<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainboardControl</class>
 <widget class="QWidget" name="MainboardControl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>-30</y>
     <width>991</width>
     <height>641</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/image/gui.png);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonSwitch1">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>90</y>
     <width>97</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Weiche </string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonSwitch2">
   <property name="geometry">
    <rect>
     <x>923</x>
     <y>460</y>
     <width>97</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Weiche </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_switch_11">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>115</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/image/arrow.png);
background: white;

</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_switch_21">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>891</x>
     <y>437</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/image/arrow.png);
background: white;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_switch_10">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>144</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/image/arrow.png);
background: white;

</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_switch_20">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>891</x>
     <y>407</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/image/arrow.png);
background: white;


</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QComboBox" name="combo_box_house_1">
   <property name="geometry">
    <rect>
     <x>954</x>
     <y>276</y>
     <width>86</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="combo_box_house_2">
   <property name="geometry">
    <rect>
     <x>1144</x>
     <y>275</y>
     <width>86</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
